sdk/compiler/damlc/lib/DA/Cli/Damlc/Command/MultiIde/Forwarding.hs

```diff
@@ -187,4 +188,7 @@ filePathFromURI miState uri =
           pure $ LSP.fromNormalizedFilePath $ vrScenarioFile vr
         "untitled:" ->
           pure $ unPackageHome $ misDefaultPackagePath miState
+        -- For the GitHub Pull Request extension, which gives paths of the form `review:commit~<hash>/path/to/the/file`
+        "review:" ->
+          pure $ joinPath $ drop 1 $ splitPath $ URI.uriPath parsedUri
```

Comment:
I'll have to double check, but I don't think its a diff, I believe its showing the old version of the file in which the review comment was placed on.
We could omit it I guess, I just noticed the path in my errors and though - thats easy to pull a path from.
I'll check

Reply:
If it's not a diff then sure, sounds like a good idea. The only problem I can think might arise is that the old Git file might map to the same filename as the current version of the file open elsewhere, and I expect that something would break somewhere - is there anything in the system that prevents that?

Reply:
Ah true, I think this would confuse the IDE, i may revert this.